great_expectations_cloud/agent/actions/run_column_descriptive_metrics_action.py

```diff
@@ -39,7 +39,7 @@ def __init__(
             data_store=CloudDataStore(self._context)
         )
         self._batch_inspector = batch_inspector or BatchInspector(
-            context, ColumnDescriptiveMetricsMetricRetriever(self._context)
+            context, [ColumnDescriptiveMetricsMetricRetriever(self._context)]
```

Comment:
Can this be covered by a test?

Reply:
We have quite a few existing tests on this action: https://github.com/great-expectations/cloud/blob/c52d4696eb2ae075d4a1d8f2d174bb2350d254cf/tests/agent/actions/test_run_column_descriptive_metrics_action.py

Reply:
good idea, ty, i will look into that after merging this just to get the fix out asap